app/models/grda_warehouse/utility.rb

```diff
@@ -117,6 +117,8 @@ def self.clear!
       tables << CasCeData::Synthetic::Assessment
     end
 
+    tables << Hmis::AutoExitConfig if RailsDrivers.loaded.include?(:hmis)
```

Comment:
Needed to prevent tests from breaking due to foreign key constraints on the auto exit configs